packages/calcite-components/src/demos/validation.html

```diff
@@ -457,6 +510,12 @@ <h1 style="margin: 0 auto; text-align: center">Form Validation</h1>
       datePicker.maxAsDate = new Date();
       datePicker.min = "2021-01-01";
 
+      // Autocomplete
+      const autocomplete = document.getElementById("framework");
+      autocomplete.addEventListener("calciteAutocompleteChange", (event) => {
+        autocomplete.inputValue = event.target.value;
```

Comment:
This is for demonstration purposes only. We don't update the inputValue when a value has been selected. This is because we don't know what data would be best to put into the input. A value may be a unique id, a description or label may be better. Its best to leave this up to the end user to decide if they want to populate the input or not. In some cases, it may be best to leave the input as is with whatever the user has typed in so they can adjust their query.